src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -789,6 +796,26 @@ def pickRSE(self, rseExpression='rse_type=TAPE\cms_type=test', rseAttribute='ddm
         choice = weightedChoice(rsesWithWeights)
         return choice
 
+    def requiresApproval(self, rse):
+        """
+        _requiresApproval_
+
+        Returns wether or not the specified RSE requires rule approval.
+        :param res: Rucio RSE name
+        :returns: True if the RSE requires approval to write (rule property)
+        """
+        
+        try:
+            attrs = self.cli.list_rse_attributes(rse)
+        except RSENotFound as exc:
```

Comment:
How about we don't throw this exception, instead just make a log error record, and return a sensible default value as you do in the line below?

Reply:
And changing my mind here, perhaps we can through the generic WMRucioException with the proper error message, instead of creating a new one.

Reply:
For this we may use `True` as default. I was not sure of what should be the response to trying to create a rule to an un-existent site. I'll make the changes as you suggest.